packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -0,0 +1,580 @@
+/**
```

Comment:
Most of the contents of this file have been ported over 1:1 from `hooks/background.js` with the modifications necessary to handle inherited values from theme.json/global styles.